# Samples require OpenSSL library on host that should not be required
diff --git a/sdk/identity/azure-identity/CMakeLists.txt b/sdk/identity/azure-identity/CMakeLists.txt
index 5a099b0e..ba8920dc 100644
--- a/sdk/identity/azure-identity/CMakeLists.txt
+++ b/sdk/identity/azure-identity/CMakeLists.txt
@@ -92,6 +92,3 @@ if (BUILD_PERFORMANCE_TESTS)
   add_subdirectory(test/perf)
 endif()
 
-if (AZ_ALL_LIBRARIES)
-  add_subdirectory(samples)
-endif()

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 173bca57..e5e4e9a0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -95,4 +95,3 @@ add_subdirectory(sdk/identity)
 add_subdirectory(sdk/keyvault)
 add_subdirectory(sdk/storage)
 add_subdirectory(sdk/template)
-add_subdirectory(samples/integration/vcpkg-keyvault)

